var/spack/repos/builtin/packages/qthreads/package.py

```diff
@@ -29,13 +29,23 @@ class Qthreads(AutotoolsPackage):
 
     homepage = "http://www.cs.sandia.gov/qthreads/"
 
-    url = "https://github.com/Qthreads/qthreads/releases/download/1.10/qthread-1.10.tar.bz2"
+    url = "https://github.com/Qthreads/qthreads/releases/download/1.18/qthread-1.18.tar.gz"
```

Comment:
You don't need this now that you have `url_for_version()`.

Reply:
I've updated the function to no longer rely on the default URL schema and removed this line.